src/pk/asn1/der/sequence/der_length_sequence.c

```diff
@@ -46,6 +52,9 @@ int der_length_sequence(ltc_asn1_list *list, unsigned long inlen,
           break;
        }
 
+       /* some items may be optional during import */
+       if (!list[i].used && list[i].optional) continue;
```

Comment:
The `der_length_sequence` (which calls `der_length_sequence_ex`) is called during decoding - see `der_decode_sequence_ex.c`.

Possible theory: it seems that during decode operation I use `optional` to indicate that it is ok if the item is missing and `used` to indicate whether it was really decoded.